### PR TITLE
Change summary, description and category, add website, bugs, repository

### PR DESCRIPTION
**Pull request by pmaier1**
Documentation is missing as we don't have anything yet.
Screenshot is still empty. Following are two screenshots that still need to be checked regarding dimensions. Please check and upload @hurradieweltgehtunter, then we can insert here and merge. Thanks!

Btw. I can't make @tobiherr a reviewer here :/

oauth_1

oauth_2

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
     <licence>AGPL</licence>
     <author>Project Seminar "sciebo@Learnweb" of the University of Münster, Thomas Müller</author>
     <version>0.1.1</version>
     <namespace>OAuth2</namespace>
-    <category>integration</category>
+    <category>security</category>
+    <website>https://github.com/owncloud/oauth2</website>
+    <bugs>https://github.com/owncloud/oauth2/issues</bugs>
+    <repository type="git">https://github.com/owncloud/oauth2.git</repository>
+    <screenshot></screenshot>
```
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: no screenshot?

Comment (pmaier1) [inline appinfo/info.xml]: Read top post :)

Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: sry :)

**Comment by hurradieweltgehtunter** [comment]
added screenshot. Description, summary etc. fine with me

**Pull request merged**